static/js/sefaria/sefaria.js

```diff
@@ -479,9 +479,10 @@ Sefaria = extend(Sefaria, {
     let refStrs = [""];
     refs.map(ref => {
       let last = refStrs[refStrs.length-1];
-      const encodedRef = encodeURIComponent(ref)
-      if (`${hostStr}${last}|${encodedRef}${paramStr}`.length > MAX_URL_LENGTH) {
-        refStrs.push(encodedRef)
+      const encodedRef = encodeURIComponent(ref);
+      const encodedFullURL = encodeURIComponent(`${hostStr}${last}|${ref}${paramStr}`);
```

Comment:
the only thing that needs to be encoded here is pipe if I understand correctly (and we need to use encodedRef instead of ref).